pyomo/solvers/plugins/solvers/SCIPAMPL.py

```diff
@@ -216,70 +277,108 @@ def _postsolve(self):
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.unknown
+             
+        # # OK
+        # maxEvaluations='maxEvaluations'                 # Exceeded maximum number of problem evaluations
+                    
         elif results.solver.message == "node limit reached":
+        
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.maxEvaluations
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # maxEvaluations='maxEvaluations'                 # Exceeded maximum number of problem evaluations
+        
         elif results.solver.message == "total node limit reached":
             results.solver.status = \
-                SolverStatus.aborted
+                SolverStatus.ok
             results.solver.termination_condition = \
                 TerminationCondition.maxEvaluations
             if len(results.solution) > 0:
                 results.solution(0).status = \
                     SolutionStatus.stoppedByLimit
+                    
+        # # OK
+        # maxEvaluations='maxEvaluations'                 # Exceeded maximum number of problem evaluations
```

Comment:
I appreciate these comments being inserted, but can they please be cleaned up a bit? I.e., one comment per line?